_submissions/round-12/1/2015-03-05-openmp-outline.md

```diff
@@ -0,0 +1,15 @@
+---
+date: 2015-03-05
+round: Round 12
+title: OpenMP Topics/Outline
+author: John Pormann
+permalink: /2015/02/
+tags:
+  - Concept Map
+  - OpenMP
+---
+
+Basic concept map for an OpenMP programming class, including major topics and
+interrelations.
+
+![http://i.imgur.com/mhdWbIq.jpg](http://i.imgur.com/mhdWbIq.jpg)
```

Comment:
The spec called for concept maps of topics in our core material, but given the week you've had, we'll take this - thanks.